CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+
+## [Unreleased]
+
+### Fixed
+* Set Azure containers public access level to `blob` instead of `container` to avoid anonymous to list the content.
```

Comment:
"to ensure anonymous users cannot list the content."

Reply:
Modified